README.md

```diff
@@ -2,95 +2,75 @@
 
 ![](https://hackmd.io/_uploads/ByWjLKAhn.png)
 
-[Discord](https://discord.gg/X7ystzGcf5) | [Website](https://evo.ninja)
-Give this repo a star if you use it! :star: 
+[Discord](https://discord.gg/X7ystzGcf5) | [Website](https://evo.ninja) | Give the repo a :star: !  
 
-## Welcome to evo.ninja
+## Welcome!
 
-**The AI that evolves in real-time.** 
+To get started using evo.ninja simply head to our [website](https://evo.ninja), or to build and run from source follow these [setup instructions](#setup).
 
-It executes scripts to achieve a goal. It is capable of using fuzzy search to find and execute any script in its library. Scripts are namespaced JavaScript functions with typed arguments and a description. If it can not find a script, it will write one itself.
+## How it works
 
-Dive in to explore the capabilities and features provided by this agent.
-Before you can run evo.ninja, ensure you have Node.js and yarn installed.
+What makes evo.ninja special is that it evolves itself in real-time, based on the tasks at hand. Evo utilizes pre-defined agent personas that are tailored to specific domains of tasks. Each iteration of evo's execution loop it will select and adopts the persona that fits the task at hand best.
```

Comment:
Is it really fair to say it "evolves" in real time? This made more sense when it wrote scripts in real time; but the persona selection make it more fair to say it "adapts" in real time. IMHO